### PR TITLE
Format may not be defined. Provide a default and only add to URL if i…

### PR DESCRIPTION
**Pull request by Dillon-Dumesnil**
…t exists

### CODE REVIEW

File: src/courseware/course/sequence/Unit.jsx [hunk 1]
```diff
@@ -57,7 +57,10 @@ function Unit({
   id,
   intl,
 }) {
-  const iframeUrl = `${getConfig().LMS_BASE_URL}/xblock/${id}?show_title=0&show_bookmark_button=0&format=${format}`;
+  let iframeUrl = `${getConfig().LMS_BASE_URL}/xblock/${id}?show_title=0&show_bookmark_button=0`;
```
Comment (Dillon-Dumesnil) [inline src/courseware/course/sequence/Unit.jsx:60]: Is this proper syntax or should it be
```
let iframeUrl;
iframeUrl = ...
if (format)...

Comment (davidjoy) [reply src/courseware/course/sequence/Unit.jsx:60]: I like the version you have here in the PR.  For some reason I don't like declaring but not initializing variables. ¯\_(ツ)_/¯  Personal preference.

File: src/courseware/course/sequence/Unit.jsx [hunk 2]
```diff
@@ -151,13 +154,14 @@ function Unit({
...
   courseId: PropTypes.string.isRequired,
-  format: PropTypes.string.isRequired,
+  format: PropTypes.string,
   id: PropTypes.string.isRequired,
   intl: intlShape.isRequired,
   onLoaded: PropTypes.func,
 };
 
 Unit.defaultProps = {
+  format: '',
```
Comment (davidjoy) [inline src/courseware/course/sequence/Unit.jsx]: I'd let this stay as `null`, which is what it's coming through as from higher up in the app.  Means it has a consistent value until it's added to the URL.

**Review by davidjoy — APPROVED** [review]
One nit, but looks good.

File: src/courseware/course/sequence/Unit.jsx (hunk 2 above)
Comment (Dillon-Dumesnil) [reply src/courseware/course/sequence/Unit.jsx]: Yeah, totally fair point. I questioned this myself. I went with the empty string so it matched the PropType, but I do like the consistency of null. Will update.

**Pull request merged**